src/main/java/com/mraof/minestuck/item/weapon/PropelEffect.java

```diff
@@ -0,0 +1,68 @@
+package com.mraof.minestuck.item.weapon;
+
+import com.mraof.minestuck.player.EnumAspect;
+import com.mraof.minestuck.player.Title;
+import com.mraof.minestuck.world.storage.PlayerSavedData;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.*;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.world.World;
+
+public class PropelEffect implements ItemRightClickEffect
+{
+	public static final PropelEffect BREATH_PROPEL = new PropelEffect(3, EnumAspect.BREATH);
+	
+	private final double velocity;
+	private final EnumAspect aspect;
+	private Title title;
```

Comment:
This field is not appropriate for this type of class. Use of it should not be necessary, and there's a high risk of unexpected behavior such as leakage between onRightClick() calls (in this case between sides). You should keep it as a local variable to avoid this.

Reply:
This is referring to the Title variable?

Reply:
Yes

Reply:
It is now a local variable, and the reason for its previous use in the field has been solved!